app/Models/Entry.php

```diff
@@ -241,20 +241,20 @@ public function content(bool $withEnclosures = true, bool $allowDuplicateEnclosu
 
 			if (self::enclosureIsImage($enclosure)) {
 				$content .= '<p class="enclosure-content"><img src="' . $elink . '" alt="" title="' . $etitle . '" /></p>';
-			} elseif ($medium === 'audio' || strpos($mime, 'audio') === 0) {
+			} elseif ($medium === 'audio' || str_starts_with((string) $mime, 'audio')) {
```

Comment:
Example of manual casting, which should really be avoided. Should be fixed higher up.